### PR TITLE
is288/service input assignment

### PR DESCRIPTION
**Pull request by ignapas**
## What do these changes do?

Adds highlighting of output port in connected components.
Some layout and code improvements.
highlighting

## Related issue number

connects to #288 
## Checklist

- [x] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [x] **Runs in the swarm**
- [ ] If you design a new module, add your user to .github/CODEOWNERS

[waffle.io]:https://waffle.io/marketing-assets/documents/waffleio_cheatsheet_v1.pdf?utm_source=blog&utm_medium=cheatsheet-ctabutton&utm_campaign=cheatsheet

### CODE REVIEW

File: services/web/client/source/class/qxapp/component/widget/inputs/NodeOutputLabel.js [hunk 1]
```diff
@@ -87,15 +91,22 @@ qx.Class.define("qxapp.component.widget.inputs.NodeOutputLabel", {
     }
   },
 
+  events: {
+    "dragStart": "qx.event.type.Data",
+    "dragEnd": "qx.event.type.Data"
+  },
+
```
Comment (odeimaiz) [inline services/web/client/source/class/qxapp/component/widget/inputs/NodeOutputLabel.js]: Are these two events used?

**Comment by oetiker** [comment]
regarding the design of the 'buttons' in 'material' design 'borders' are not very popular ... you should rather work with differences in color and animations of the buttons on mouse over see the tool bar buttons

**Comment by ignapas** [comment]
> regarding the design of the 'buttons' in 'material' design 'borders' are not very popular ... you should rather work with differences in color and animations of the buttons on mouse over see the tool bar buttons

which buttons are you referring?

File: services/web/client/source/class/qxapp/component/widget/inputs/NodeOutputLabel.js (hunk 1 above)
Comment (ignapas) [reply services/web/client/source/class/qxapp/component/widget/inputs/NodeOutputLabel.js]: no, they are not. good call...

File: services/web/client/source/class/qxapp/component/widget/inputs/NodeOutputLabel.js [hunk 2]
```diff
@@ -88,14 +92,16 @@ qx.Class.define("qxapp.component.widget.inputs.NodeOutputLabel", {
   },
 
   members: {
+    __portId: null,
```
Comment (odeimaiz) [inline services/web/client/source/class/qxapp/component/widget/inputs/NodeOutputLabel.js]: prop?

Comment (ignapas) [reply services/web/client/source/class/qxapp/component/widget/inputs/NodeOutputLabel.js]: I am using it only internally so I think for the moment there's no need to generate any accessors/setters

**Pull request merged**